setup-scripts/health-checker.py

```diff
@@ -114,55 +114,86 @@ async def check_health():
+    critical_checks_failed = 0
+
+    verbose_checks_total = 0
+    verbose_checks_failed = 0
 
-    # Check the health records.
-    passed_checks = 0
-    failed_checks = 0
-    failed_critical = 0
     failed_records = []
+    failed_records_file = None
+
     time_limit_unaware = datetime.now() - timedelta(hours=CHECK_HOURS)  # local time
     time_limit = time_limit_unaware.astimezone(get_localzone())  # time with time zone
-    for rec in res.json():
-        time_unaware = datetime.strptime(rec['date'], '%Y-%m-%dT%H:%M:%S.%fZ')  # time in UTC
+
+    for critical in json_critical:
+        time_unaware = datetime.strptime(critical['date'], '%Y-%m-%dT%H:%M:%S.%fZ')  # time in UTC
         time = pytz.utc.localize(time_unaware)  # time with time zone
         if time < time_limit:
             continue
         bad = False
-        for check in rec['checks']:
+        for check in critical['checks']:
+            critical_checks_total += 1
             if check['up'] == False:
+                critical_checks_failed += 1
                 bad = True
-                failed_checks += 1
-                if check['critical']:
-                    failed_critical += 1
         if bad:
-            # We append the entire record, so we can get the full context.
-            failed_records.append(rec)
-        passed_checks += 1
-
-    checks = passed_checks + failed_checks
-    if len(failed_records) > 0:
-        message = "Found {}/{} failed checks ({} critical) over the last {} hours!".format(failed_checks, checks,
-                                                                                           failed_critical, CHECK_HOURS)
-        file = discord.File(io.BytesIO(json.dumps(failed_records, indent=2).encode()), filename="failed_checks.log")
-        notifyTeam = failed_critical > 0
-        await send_msg(client, message, file=file, force_notify=notifyTeam)
-        return
+            failed_records.append(critical["checks"])
+
+    for verbose in json_verbose:
+        time_unaware = datetime.strptime(verbose['date'], '%Y-%m-%dT%H:%M:%S.%fZ')  # time in UTC
+        time = pytz.utc.localize(time_unaware)  # time with time zone
+        if time < time_limit:
+            continue
+        bad = False
+        for check in verbose['checks']:
+            verbose_checks_total += 1
+            if check['up'] == False:
+                verbose_checks_failed += 1
+                bad = True
+        if bad:
+            failed_records.append(verbose["checks"])
+
+    ################################################################################
+    ################ create a message
+    ################################################################################
+
+    message = ""
+    force_notify = False
+
+    if json_check["disabled"]:
+        message += "__Portal manually disabled!__ "
+        force_notify = True
+    elif res_check.status_code is not requests.codes.ok:
+        message += "__Portal down!!!__ "
```

Comment:
I'd make this all caps. Or at least `DOWN!!!`. :)

Reply:
it's going to be underlined and will produce a discord mention so it should be fine :)

Reply:
Ok, urgent enough. :)))